### DISCUSSION TITLE
BUG: No Option for Publishing page to Web

### DISCUSSION THREAD

**Discussion by deepakgohil9** [GitbookIO/community#630] [0. General bugs and questions]
I'm trying to publish my Gitbook space to the web, but I'm unable to find the "Publish to the web" option mentioned in the documentation at [https://docs.gitbook.com/published-documentation/share/space-publishing](https://docs.gitbook.com/published-documentation/share/space-publishing).

![image](https://github.com/GitbookIO/community/assets/93052295/975f7664-c70a-41d7-8b15-23063ed9551a)

I've searched through the platform and reviewed the documentation, but I can't locate any feature related to web publishing.

Could you please clarify how to publish a Gitbook space to the web, or if this functionality is no longer available?

**Comment by SamyPesse** [comment]
It's still possible to publish a space to the web. Your account seems to be part of a segment of users that have access to the new feature to do so: Docs site. This feature is not yet documented in our docs.

To publish your space, you can create a Docs site from the "Docs sites" entry in the sidebar; and then select your space as the source of content.

Let us know if you have feedback about this flow.

**Reply by attex** [reply]
@SamyPesse it's not. 

It just became a pro feature.

I also have this new feature but now I'm not able to publish anything at all.

<img width="589" alt="Screenshot 2024-06-04 at 7 28 04 PM" src="https://github.com/GitbookIO/community/assets/9304272/f8dc1372-fa55-4a91-b7cd-ce6ab6048940">

**Reply by 398-Studios** [reply]
> @SamyPesse it's not.
> 
> It just became a pro feature.
> 
> I also have this new feature but now I'm not able to publish anything at all.
> <img alt="Screenshot 2024-06-04 at 7 28 04 PM" width="589" src="https://private-user-images.githubusercontent.com/9304272/336549705-f8dc1372-fa55-4a91-b7cd-ce6ab6048940.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1NDU2NTMsIm5iZiI6MTcxNzU0NTM1MywicGF0aCI6Ii85MzA0MjcyLzMzNjU0OTcwNS1mOGRjMTM3Mi1mYTU1LTRhOTEtYjdjZC1jZTZhYjYwNDg5NDAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMjM1NTUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MWU3ZWFjNzU2OWFlM2FhZWZlZGJjMzZmZGQ4YzQ5NjZiOTdhMmU5ZTJiOWVmOTc0NGFmZWI2YWUyN2RlNmE3YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.0_h86LlWDl1l-SeUh2KYvJvoe9-d1jNvfI3buMtCaZ0">

if you havent fixed it already the reason its showing as a pro is required to publish is because there are some pro feautures enabled by default. below are the 2 features I had to disable to be able to publish a docs site. Was annoying as I thought they had actually locked it behind pro. 
![image](https://github.com/GitbookIO/community/assets/51074700/2de069ac-00b0-4c0c-b857-e7618b080587)

**Reply by attex** [reply]
Thank you! 
Issue solved.

**Comment by jhoney124** [comment]
For a reliable way to track your packages, use TraceShipments.com. With TraceShipments.com, you can [trace your order](https://traceshipments.com/) easily, as it consolidates information from numerous courier companies into one user-friendly platform. With robust analytics and reporting tools, TraceShipments.com also helps businesses enhance their delivery performance and logistics management, making package tracking more efficient and convenient for both individuals and companies.

**Comment by meshalA1** [comment]
حسناااااااا يمكنك الأتصال علي وافعل كل ماتريد أنا بنفعي وأما زيادت الضرر إللي لم ارتح منه 966536590821 اخبرني ماهو رقم هاتفك لأنني لن أستجيب لكل شخص يدعي المسكنه والحزن ما انا لأيرف قلبي ما ترف شواربكم وراء ماتفعلونه من رذيله وسرقه ‫أُرسلت من الـ iPhone‬‫في ٢٧‏/٠٦‏/٢٠٢٤ الساعة ١١:٤٤ ص، كتب/كتبت ‏jhoney124 ***@***.***>:‬﻿
For a reliable way to track your packages, use TraceShipments.com. With TraceShipments.com, you can trace your order easily, as it consolidates information from numerous courier companies into one user-friendly platform. With robust analytics and reporting tools, TraceShipments.com also helps businesses enhance their delivery performance and logistics management, making package tracking more efficient and convenient for both individuals and companies.
